code/nrf-connect/prstlib/src/adc.c

```diff
@@ -0,0 +1,159 @@
+#include "prstlib/adc.h"
+
+#include <math.h>
+#include <zephyr/drivers/adc.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/pwm.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/zephyr.h>
```

Comment:
I'm using the `main` branch of NCS, and with it, anoher warning:

`#warning "<zephyr/zephyr.h> is deprecated, include <zephyr/kernel.h> instead"`

Reply:
Tks, done (also for other files in this PR).